src/main/java/com/example/App.java

```diff
@@ -12,6 +12,9 @@ public App() throws IOException {
         if (false) {
             // another useless block
         }
+        if (false) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Change this condition so that it does not always evaluate to "false" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://docker.for.mac.localhost:9000/coding_rules#rule_key=squid%3AS2583)